src/Provider/PointCloudProvider.js

```diff
@@ -138,7 +139,7 @@ function loadPointFile(layer, url) {
         if (layer.metadata.customBinFormat) {
             return PotreeCinParser.parse(ab).then(result => addPickingAttribute(result));
         } else {
-            return PotreeBinParser.parse(ab).then(result => addPickingAttribute(result));
+            return PotreeBinParser.parse(ab, layer.metadata.pointAttributes).then(result => addPickingAttribute(result));
```

Comment:
I'm not sure yet, that's why I did an adhoc call here. I think this question should be tackled when/if we have a parser registry mechanism for pointclouds, WDYT? Not yet sure how this will be done though (there are probably things that can be unified with 3dTiles pnts format I guess).